examples/node_example/testem.json

```diff
@@ -1,9 +1,10 @@
 {
-    "launchers": {
-        "Node": {
-            "command": "mocha tests.js"
-        }
-    },
-    "launch_in_dev": [ "Node" ],
-    "launch_in_ci": [ "Node" ]
+  "launchers": {
+    "Node": {
+      "command": "mocha tests.js -R tap",
+      "protocol": "tap"
```

Comment:
Removed it. I wasn't sure if broken output was intended or just something got mangled along the way.